src/js/Reducers/SettingsReducer.js

```diff
@@ -161,6 +164,32 @@ export const SettingsReducer = (state, action) => {
 
       return state;
 
+    case 'SETTINGS_TOGGLE_CONFIG':
+      localStorage.setItem('use_lightdm_conf', !state.settings.use_lightdm_conf);
+
+      var dir              = FileOperations.getWallpaperDirectory();
+      var wallpaper        = Settings.requestSetting('wallpaper', 'space-1.jpg');
```

Comment:
Why would the user want their wallpaper reset when toggling config usage?

Reply:
The wallpaper will only reset to `space-1.jpg` only if the localStorage's wallpaper value is either null or undefined (see 7960b33b821e80e01e589f863c3949cf2ee82141 src/js/Logic/Settings.js ln 29)

Reply:
I'm referring to the general idea of manually re-implementing the wallpaper switcher behavior here and resetting the wallpaper whenever the source changes, not changing to the default you've specified. I'm dubious in general to the idea of unsynced configurations implemented in this fashion.

Reply:
Because the wallpaper is segregated from the other settings. IMO to make the code cleaner a massive refactor needs to happen. Actions and action types needs to be separated from the container and reducer, wallpaper settings should not be separated in the settings state, while keeping the wallpaper switcher's state on a local state.